docs/wip_glassbox/entity-extension.md

```diff
@@ -133,3 +133,18 @@ So in the project, we can change the factory to produce extended entity data ins
   * add your extended entity into `$entityExtensionMap` in the `setUp()` method
 
 *Tip: to see how it works in practice check out `\Tests\ShopBundle\Functional\EntityExtension\EntityExtensionTest` that tests end-to-end extensibility of `Product`, `Category` and `OrderItem`.*
+
+## Limitations
+
+### OrderItem
+
+`OrderItem` is a bit special entity, because it is not created from `OrderItemData`, it is created from different sources like from product itself.
+All creations are done by [`Order`](/packages/framework/src/Model/Order/Order.php) entity only, where you can check that it really make sense to not create the `OrderItem` from a data object.
+
+If you need to extend the `OrderItem` by a new field, for example, an ID from an external system, you'll have to fill this field after the `Order` is created by a new public method on `Order` class.
+And then if you'll need to be able to edit this field from the administration, you'll have to override `edit` method of `Order` entity and solve setting this new field there.
+
+The other way of data from - from `OrderItem` to `OrderItemData` is standard.
+So if you extend `OrderItem` in a standard fashion, add setting a new field in extended `OrderItemDataFactory` then `OrderItemData` object will contain the correct value from the `OrderItem` object.
+
+Creating a new type of `OrderItem` is possible and doesn't cause problems because the new type is completely in your hands.
```

Comment:
doesn't -> does not

Reply:
Why? Do we have some rule to not use contractions? For me, it sounds more natural, but I'm eager to hear different opinions

Reply:
It is more formal, it is only reason